.github/workflows/query-docker.yml

```
@@ -0,0 +1,94 @@
+name: "Query-to-docker"
+on:
+  release:
+    types:
+      - published
+  workflow_dispatch:
+    inputs:
+      isLatest:
+        description: "Add latest tag"
+        default: "true"
+        required: true
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    outputs:
+      changes_found: ${{ steps.check_changes.outputs.changes_found }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+      - name: Check for package changes and commit message
+        id: check_changes
+        run: |
+          if [[ "${{ github.event_name }}" == "release" ]]
+          then
+            TAG_NAME=${GITHUB_REF#refs/tags/}
+            if [[ $TAG_NAME == "query-subgraph/"* ]] 
+            then
+              echo "changes_found=true" >> "$GITHUB_OUTPUT"
+            else
+              echo "changes_found=false" >> "$GITHUB_OUTPUT"
+            fi
+          else
+            echo "changes_found=true" >> "$GITHUB_OUTPUT"
+          fi
+
+  query-build-push-docker-subquery:
+    needs: check
+    if: needs.check.outputs.changes_found == 'true'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 100
+          token: ${{ secrets.REPO_TOKEN }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+          username: subquerynetwork
+          password: ${{ secrets.SQ_DOCKERHUB_TOKEN }}
+
+      ## query
+      - name: Get updated query version
+        id: get-query-version
+        run: |
+          sh .github/workflows/scripts/queryVersion.sh
+
+      - run: yarn
+      - name: build
+        run: yarn build
+
+      - name: Build and push
+        if: github.event_name == 'workflow_dispatch' && github.event.inputs.isLatest == 'false'
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          push: true
+          platforms: arm64,amd64
+          file: Dockerfile
+          tags: subquerynetwork/subgraph-query:v${{ steps.get-query-version.outputs.QUERY_VERSION }}
```

```suggestion
          tags: subquerynetwork/query-subgraph:v${{ steps.get-query-version.outputs.QUERY_VERSION }}
```

This should match the repo name